ci-bench/src/main.rs

```diff
@@ -210,31 +210,31 @@ fn all_benchmarks() -> anyhow::Result<Vec<Benchmark>> {
 static ALL_BENCHMARK_PARAMS: &[BenchmarkParams] = &[
     BenchmarkParams::new(
         KeyType::Rsa,
-        rustls::cipher_suite::TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+        cipher_suite::TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
```

Comment:
For the purpose of this file and other test/example modules in our repository, why not import the cipher suite names directly at the top of the module? (As in `use rustls::crypto::ring::cipher_suite::{TLS13_AES_128_GCM_SHA256, ..};`.)

Reply:
There are instances of this elsewhere (arguably tests that are interested in multiple individual suites should just `use ...::cipher_suite::*;` and be done with it for brevity's sake.

But in this particular case, this has fallen away with refactoring the ci-bench code to benchmark ring and aws-lc-rs in one program.